typhos/tweakable.py

```diff
@@ -0,0 +1,88 @@
+"""
+Tweakable value widget.
+
+Variety support pending:
+- everything
```

Comment:
Is there some complexity threshold by which a widget gets its own file vs being in `typhos.widgets`?

Reply:
Not _really_. I prematurely started moving these out to separate files, expecting that with more metadata support they would balloon much larger than they currently are. 

It'd probably be advisable to move widget source around in the future (a topic for another issue, I suppose).